scss/settings/_settings.scss

```diff
@@ -73,6 +73,7 @@ $global-radius: 0;
 $global-text-direction: ltr;
 $global-flexbox: false;
 $print-transparent-backgrounds: true;
+$contrast-warnings: true;
```

Comment:
Good to know. I haven't actually done the build process for this. How is the settings file created? I'm assuming by adding the variable in the *_color.scss* file, it will automatically be generated in the *_settings.scss* file? While I have used Foundation extensively, this is my first attempt at contributing 😄